tests/test_benchmark.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from os import path, listdir
+from conftest import skipif
+from devito.exceptions import CompilationError
+from devito import configuration
+from subprocess import CalledProcessError, check_call
+
+
+pytestmark = skipif(['yask', 'ops'])
+
+
+@pytest.mark.parametrize('mode', ['bench'])
+@pytest.mark.parametrize('problem', ['acoustic', 'tti', 'elastic', 'viscoelastic'])
+def test_bench(mode, problem):
+    """
+    Test combinations of modes x problems and created .json filename.
+    Testing for both OpenMP enabled or disabled.
+    """
+    if configuration['openmp']:
+        nthreads = configuration['platform'].cores_physical
+    else:
+        nthreads = 1
+
+    command_bench = ['python', 'benchmarks/user/benchmark.py', mode, '-P', problem,
+                     '-d', '40', '40', '40', '--tn', '20', '-x', '1']
+    try:
+        check_call(command_bench)
+    except CalledProcessError as e:
+        raise CompilationError('Command "%s" return error status %d. '
+                               'Unable to compile code.\n' %
```

Comment:
and most importantly, what makes you think that's due to an error with compilation?

Reply:
They are just leftovers from a pytest template.